recipes/meme/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#
+MEME_ETC_DIR=${PREFIX}/etc
+cpanm YAML
+cpanm HTML::PullParser
```

Comment:
It would be better to use these modules as dependencies and not install them via `cpanm`.